kits/ts/simple/main.py

```diff
@@ -0,0 +1,33 @@
+from subprocess import Popen, PIPE
+import atexit
+import os
+agent_processes = [None, None]
+def cleanup_process():
+    global agent_processes
+    for proc in agent_processes:
+        if proc is not None:
+            proc.kill()
+def js_agent(observation, configuration):
+    """
+    a wrapper around a js agent
+    """
+    global agent_processes
+
+    agent_process = agent_processes[observation.player]
+    ### Do not edit ###
+    if agent_process is None:
+        cwd = os.path.dirname(configuration["__raw_path__"])
+        agent_process = Popen(["node", "dist/main.js"], stdin=PIPE, stdout=PIPE, cwd=cwd)
```

Comment:
awesome! this looks great

Reply:
I am not a 100% sure what this does. Even if i have a different path instead of `dist/main.js`, `npx lux-ai-2021 dist/main.js dist/main.js` works.

Reply:
don't worry about this. This is for something else but needed